pkg/apis/allocation/v1/gameserverallocation.go

```diff
@@ -583,6 +583,12 @@ func (gsa *GameServerAllocation) ApplyDefaults() {
 		gsa.Spec.Scheduling = apis.Packed
 	}
 
+	for i := range gsa.Spec.Priorities {
```

Comment:
Please add tests to:

https://github.com/googleforgames/agones/blob/a1a1cd956a19e4e97fd67f6c0ad06da4259273ab/pkg/apis/allocation/v1/gameserverallocation_test.go#L31

Reply:
Can we also please add a comment here:

https://github.com/googleforgames/agones/blob/986ed9b98f9e830e131f73c9347d17d1eb05b32f/pkg/apis/agones/v1/common.go#L186

That the default is `Ascending`.

Reply:
MR updated following your comments @markmandel 🙂